op-challenger/fault/monitor.go

```diff
@@ -78,7 +79,15 @@ func (m *gameMonitor) progressGames(ctx context.Context) error {
 			m.logger.Error("Error while progressing game", "game", game.Proxy, "err", err)
 			continue
 		}
-		player.ProgressGame(ctx)
+		done := player.ProgressGame(ctx)
+		if done {
```

Comment:
Is this done to make it explicit what the return value is indicating here (the game being "done")?

If it's not important, could probably just make this `if player.ProgressGame(ctx) { ...`

Reply:
Yeah I think the named variable is useful for clarity.  `ProgressGame` doesn't tell you anything about what the boolean it returns means so it seems useful to name it.